### PR TITLE
Fix #169 Go over ENTIRE KB and make sure the documentation is accurate

### PR DESCRIPTION
**Pull request by EugeneButusov**

### CODE REVIEW

File: .idea/modules.xml [hunk 1]
```diff
@@ -3,6 +3,7 @@
   <component name="ProjectModuleManager">
     <modules>
       <module fileurl="file://$PROJECT_DIR$/.idea/knowledge-base.iml" filepath="$PROJECT_DIR$/.idea/knowledge-base.iml" />
+      <module fileurl="file://$PROJECT_DIR$/knowledge-base.iml" filepath="$PROJECT_DIR$/knowledge-base.iml" />
```
Comment (borisspektor) [inline .idea/modules.xml]: @EugeneButusov why is this here?

File: src/documents/cocos2dx/cpp/GrowSpend_GettingStarted.html.md [hunk 2]
```diff
@@ -330,10 +330,13 @@ bool AppDelegate::applicationDidFinishLaunching() {
 	/** Set up and initialize Store and Profile **/
 	ExampleAssets *assets = ExampleAssets::create();
 
-	__Dictionary *storeParams = __Dictionary::create();
-	storeParams->setObject(__String::create("ExamplePublicKey"), "androidPublicKey");
+    soomla::CCSoomlaStoreConfigBuilder *storeConfig
+       = soomla::CCSoomlaStoreConfigBuilder::create();
+    storeConfig
+     ->setGpConfiguration(soomla::CCSoomlaStoreGpConfigBuilder::create()
```
Comment (borisspektor) [inline src/documents/cocos2dx/cpp/GrowSpend_GettingStarted.html.md]: @EugeneButusov seems there's a `)` missing here

File: src/documents/cocos2dx/cpp/GrowUltimate_GettingStarted.html.md [hunk 3]
```diff
@@ -361,11 +361,19 @@ grow::CCGrowSync::initShared(modelSync, stateSync);
 /** Set up and initialize Store and Profile **/
 ExampleAssets *assets = ExampleAssets::create();
 
-__Dictionary *storeParams = __Dictionary::create();
-storeParams->setObject(__String::create("ExamplePublicKey"), "androidPublicKey");
+soomla::CCSoomlaStoreConfigBuilder *storeConfig
+   = soomla::CCSoomlaStoreConfigBuilder::create();
+storeConfig
+  ->setGpConfiguration(soomla::CCSoomlaStoreGpConfigBuilder::create()
```
Comment (borisspektor) [inline src/documents/cocos2dx/cpp/GrowUltimate_GettingStarted.html.md]: @EugeneButusov `)`

File: src/documents/cocos2dx/cpp/Grow_FraudProtection.html.md [hunk 4]
```diff
@@ -15,20 +15,24 @@ platform: 'cocos2dx'
 
 <div class="info-box">General information about Fraud Protection available in this [article](/university/articles/Grow_FraudProtection).</div>
 
-In order to turn on Fraud Protection in cocos2d-x, you need to pass additional params to `storeParams` depending on a
+In order to turn on Fraud Protection in cocos2d-x, you need to enable additional params to your Store config depending on a
```
Comment (borisspektor) [inline src/documents/cocos2dx/cpp/Grow_FraudProtection.html.md]: @EugeneButusov `to your Store` -> `in your Store`

File: src/documents/cocos2dx/cpp/Grow_Sync.html.md [hunk 5]
```diff
@@ -340,10 +340,13 @@ bool AppDelegate::applicationDidFinishLaunching() {
 	/** Set up and initialize Store and Profile**/
 	ExampleAssets *assets = ExampleAssets::create();
 
-	__Dictionary *storeParams = __Dictionary::create();
-	storeParams->setObject(__String::create("ExamplePublicKey"), "androidPublicKey");
+	soomla::CCSoomlaStoreConfigBuilder *storeConfig
+           = soomla::CCSoomlaStoreConfigBuilder::create();
+    storeConfig
+      ->setGpConfiguration(soomla::CCSoomlaStoreGpConfigBuilder::create()
+        ->setAndroidPublicKey("ExamplePublicKey");
```
Comment (borisspektor) [inline src/documents/cocos2dx/cpp/Grow_Sync.html.md]: @EugeneButusov  )

File: src/documents/soomla/unity/samples/tune.html.md [hunk 6]
```diff
@@ -229,7 +229,7 @@ public class TuneSoomlaInsightsScript : MonoBehaviour {
         MATBinding.MeasureSession();
 
         // Add event listeners - Make sure to set the event handlers before you initialize
-        HighwayEvents.OnInsightsInitialized += OnSoomlaInsightsInitialized;
+        HighwayEvents.OnGrowInsightsInitialized += OnSoomlaInsightsInitialized;
```
Comment (borisspektor) [inline src/documents/soomla/unity/samples/tune.html.md]: @EugeneButusov might as well change the other methods with soomla to be more updated, not just leave `OnSoomlaInsightsInitialized` as it is

File: .idea/modules.xml (hunk 1 above)
Comment (EugeneButusov) [inline .idea/modules.xml]: @borisspektor, my fault, have some difficulties, my IDE didn't show changes in this file.

**Comment by EugeneButusov** [comment]
@borisspektor, I processed your feedback.
